### DISCUSSION TITLE
NewPipe forks

### DISCUSSION THREAD

**Discussion by TruongTanLuc201270** [TeamNewPipe/NewPipe#11113] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

In warning message: "PUTTING NEWPIPE, OR ANY FORK OF IT, INTO THE GOOGLE PLAY STORE VIOLATES THEIR TERMS AND CONDITIONS."
- PipePipe have NicoNico and BiliBili: https://github.com/InfinityLoop1308/PipePipe
- Tubular have SponsorBlock and ReturnYouTubeDislike: https://github.com/polymorphicshade/Tubular
-  NewPipeSponsorBlock only SponsorBlock but no longer development: https://github.com/gilbsgilbs/NewPipeSponsorBlock
Does all of them are NewPipe requested?

**Comment by opusforlife2** [comment]
> Does all of them are NewPipe requested?

Can't understand. Try using a translation website.
